src/crypto.rs

```diff
@@ -0,0 +1,46 @@
+pub use std::convert::TryFrom;
+
+pub const KEY_SIZE: usize = 32;
+
+#[derive(Clone, Copy, Debug, Default, Eq, Ord, PartialEq, PartialOrd)]
```

Comment:
Do you think it's safe to implement `Default` for this type? (I'm not sure myself.) I could see future changes accidentally forgetting to initialize fields with this type properly.

One alternative would be to use a `Key::zero()` function to be explicit.

Reply:
I frankly don’t remember now why I chose to implement it, but an explicit zeroing sounds better. I’ll hack that in a bit later.

Reply:
Changed.